model-optimizer/mo/utils/ir_reader/extenders/range_extender.py

```diff
@@ -0,0 +1,28 @@
+"""
```

Comment:
Add a file `model-optimizer/mo/utils/ir_reader/extenders/range_extender.py` to the BOM.